### ISSUE TITLE
FR: Desktop: Dark theme for desktop

### ISSUE THREAD

**Issue by gerroon** [laurent22/joplin#617]
I like the dark theme for the Android as a user. It would be nice to get dark/grey theme for the desktop version as well.

**Comment by tessus** [comment]
Earlier comment from Laurent: _In general, for all theme-related requests, the issue is the desktop app currently supports only one theme. Once support for a second one is done, it will be possible to add any kind of theme._

Duplicate of #93
